exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -37,54 +37,24 @@
 import io.opentelemetry.sdk.metrics.data.MetricData;
 import io.opentelemetry.sdk.metrics.data.MetricDataType;
 import io.opentelemetry.sdk.metrics.data.SumData;
-import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
-import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class MetricTranslator {
+/** Utility methods to translate metrics from OTEL to GCM format. */
+public final class MetricTranslator {
 
   private static final Logger logger = LoggerFactory.getLogger(MetricTranslator.class);
 
-  static final String DESCRIPTOR_TYPE_URL = "custom.googleapis.com/OpenTelemetry/";
+  static final String DESCRIPTOR_TYPE_URL = "workload.googleapis.com/";
```

Comment:
is this configurable, or should it be?  We can tackle that at a later point if needed.

Reply:
It is not.  We can tackle later I think, as it wasn't configurable before.